opentelemetry-sdk/src/trace/sampler/jaeger_remote/rate_limit.rs

```diff
@@ -53,10 +52,12 @@ impl LeakyBucket {
                         false
                     }
                 }
-                Err(_) => {
-                    opentelemetry::global::handle_error(TraceError::Other(
-                        "jaeger remote sampler gets rewinded timestamp".into(),
-                    ));
+                Err(err) => {
+                    opentelemetry::otel_debug!(
```

Comment:
`use opentelemetry::otel_debug` to be consistent?

Reply:
done.